metadata.yml

```diff
@@ -6,13 +6,33 @@ rules:
 mutating: false
 contextAware: false
 annotations:
-  io.kubewarden.policy.title: policy-name
-  io.kubewarden.policy.description: Short description
-  io.kubewarden.policy.author: policy-author
-  io.kubewarden.policy.url: https://github.com/yourorg/policy-name
-  io.kubewarden.policy.source: https://github.com/yourorg/policy-name
+  io.kubewarden.policy.title: psp-volumes
+  io.kubewarden.policy.description: PSP that controls usage of volumes
```

Comment:
Same here

Reply:
Done in https://github.com/kubewarden/psp-volumes-policy/pull/1/commits/472c59059969cf5688bdf811c7634cbe2e353a41.